### PR TITLE
1 create new space

### LINKED ISSUE #1: Create new space
```
As a User
So that I can advertise 
I want to be able to create a space
```

### PR DESCRIPTION
**Pull request by RPiper93**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,10 @@
 
 ###Implemented User Stories
 
+```
+As a User
+So that I can advertise
+I want to be able to create a space
+```
+:construction:
```
Comment (jaxdid) [inline README.md]: Remove this if complete :)

File: app/views/spaces.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<html>
+<head>
```
Comment (jaxdid) [inline app/views/spaces.erb]: Maybe extract this to a layout file?

File: app/views/spaces_new.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<html>
+<head>
```
Comment (jaxdid) [inline app/views/spaces_new.erb]: Extract to layout file

**Comment by tcpickard94** [comment]
:shipit:

**Pull request merged**